packages/node_modules/@ciscospark/react-component-adaptive-card/src/adaptiveCard.scss

```diff
@@ -0,0 +1,271 @@
+@import '@momentum-ui/core/scss/momentum-ui';
+@import "@ciscospark/react-component-activity-text/src/styles";
+
+.activity-item--adaptive-card {
```

Comment:
Is there a way to tests this styles? it's hard to know why certain values or properties are changed

Reply:
scss is for the elements/properties of adaptive card .Snapshot contains styles for those properties  in a newly added test